proto/tendermint/types/params.proto

```diff
@@ -32,6 +33,23 @@ message BlockParams {
   int64 time_iota_ms = 3;
 }
 
+// VoteInfo
+message ChainLockParam {
+  // Note: must be greater than 0
+  uint32 chain_lock_height = 1;
```

Comment:
These names clobber a bit. Consider removing `chain_lock` prefixes from the fields.